include/net-snmp/library/snmp_api.h

```diff
@@ -529,12 +529,6 @@ struct netsnmp_transport_s;
      * See snmp_debug.h and snmp_debug.c instead.
      */
 
-    NETSNMP_IMPORT
-    void            snmp_set_do_debugging(int);
-    NETSNMP_IMPORT
-    int             snmp_get_do_debugging(void);
-
-
```

Comment:
Please keep these declarations and remove the declarations from include/net-snmp/output_api.h instead.

Reply:
Done.